examples/dda_demo/scripts/060_build_demo_functions.sh

```diff
@@ -2,8 +2,8 @@
 # SPDX-FileCopyrightText: © 2024 Siemens AG
 # SPDX-License-Identifier: MIT
 
-# Delete old function wasm files
-rm -f $(find ../functions -name "*.wasm")
-
 # Build all dda demo functions
-for i in $(find ../functions -name function.json) ; do ../../../target/debug/edgeless_cli function build $i ; done
+dda_demo_functions="check_temperature move_arm"
+for dda_demo_function in $dda_demo_functions ; do
```

Comment:
should we keep the delete to keep the behaviour consistent? i.e. not to hide any errors during function build.

Reply:
OK, it makes sense.